src/main/java/com/github/jnidzwetzki/bitfinex/v2/BitfinexApiBroker.java

```diff
@@ -611,43 +629,24 @@ private void handleChannelDataArray(final JSONArray message, final BitfinexStrea
 	 * @return
 	 */
 	public boolean isTickerActive(final BitfinexTickerSymbol symbol) {
-		return getChannelForSymbol(symbol) != -1;
+		return getChannelForSymbol(symbol) != null;
 	}
 
 	/**
 	 * Find the channel for the given symbol
 	 * @param symbol
 	 * @return
 	 */
-	public int getChannelForSymbol(final BitfinexStreamSymbol symbol) {
+	private Integer getChannelForSymbol(final BitfinexStreamSymbol symbol) {
```

Comment:
What do you think about `Optional<Integer>` as return type? This might prevent some Null-Pointer Exceptions and forces the caller to check the result.

Reply:
I am a big fan of Optional, and functional programming in general - but, since it's internal method and only we use it, makes no real difference -- I will refactor this method out in future PRs anyway